src/main/resources/application.properties

```diff
@@ -1,5 +1,9 @@
 ##DB props
 
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
```

Comment:
slightly nit picky - put your related properties together (theres other jpa properties at the bottom)